### PR TITLE
Test: Ipv6 name servers

### PR DESCRIPTION
**Pull request by Xatrekak**
## Change Summary

Test: Molecule test for IPv6 inputs with ip_name_servers

## Related Issue(s)
#3715

## Component(s) name
`arista.avd.eos_designs_unit_tests`

## How to test
`molecule converge -s  eos_cli_config_gen -- --limit dns-ntp`
`molecule converge -s eos_designs_unit_tests -- --limit AVD_LAB`

### Repository Checklist
- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

**Comment by jerryatarista** [comment]
LGTM

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ipv6_input_ip_name_servers.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+type: l2leaf
+
+l2leaf:
+  defaults:
+
+ip_name_servers:
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ipv6_input_ip_name_servers.yml]: This data model is for `eos_cli_config_gen`. Test coverage should be added here: https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dns-ntp.yml#L11

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ipv6_input_ip_name_servers.yml]: For `eos_designs` the data model is  `name_servers` and a test should be added here:
https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AVD_LAB.yml#L34.
But for this data model the description must be updated in the schema fragment: https://github.com/aristanetworks/avd/blob/devel/ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/name_servers.schema.yml#L16

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ipv6_input_ip_name_servers.yml (hunk 1 above)
Comment (Xatrekak) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/host_vars/ipv6_input_ip_name_servers.yml]: Hey Claus thanks for the feedback. I have removed the test I originally created and added to the tests you suggested.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM. Thanks

**Comment by jerryatarista** [comment]
LGTM

**Review by carlbuchmann — APPROVED** [review]
LGTM

**Pull request merged**